lib/solarwinds_apm/support/otlp_endpoint.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+module SolarWindsAPM
+  # OTLPEndPoint
+  class OTLPEndPoint
+    SW_ENDPOINT_REGEX = /^apm\.collector\.([a-z]{2}-\d{2})\.([^.]+)\.solarwinds\.com$/
+    DEFAULT_OTLP_ENDPOINT = 'https://otel.collector.na-01.cloud.solarwinds.com:443'
+    OTEL_OTLP_ENDPOINT_REGEX = %r{^https://otel\.collector\.([a-z0-9-]+)\.([a-z0-9-]+)\.solarwinds\.com:\d+/v1/(logs|metrics|traces)$}
```

Comment:
can we call this SWO_OTLP_SIGNAL_ENDPOINT_REGEX, are we sure the port will always be specified, and it doesn't seem you actually need the capture groups.

Reply:
Updated.